lib/ansible/parsing/quoting.py

```diff
@@ -25,7 +25,19 @@ def is_quoted(data):
 
 
 def unquote(data):
-    ''' removes first and last quotes from a string, if the string starts and ends with the same quotes '''
+    """Removes first and last quotes from a string, if the string starts and
+    ends with the same quotes.
+
+    @param data: The string to modify
+    @type data:  str
+
+    @return: The string without quotes
+    @rtype : str
+    """
+    if not isinstance(data, str) or not isinstance(data, bytes):
```

Comment:
This should use the compat types provided by six which is accessible from `ansible.module_utils.six` see https://six.readthedocs.io/#constants

Reply:
It's fix, thanks.